retroshare-gui/src/gui/MainWindow.cpp

```diff
@@ -651,10 +653,42 @@ void MainWindow::createTrayIcon()
     trayIcon->setContextMenu(trayMenu);
     trayIcon->setIcon(QIcon(IMAGE_NOONLINE));
 
+#if defined(Q_OS_DARWIN)
+    createMenuBar();
+#endif
+
     connect(trayIcon, SIGNAL(activated(QSystemTrayIcon::ActivationReason)), this, SLOT(toggleVisibility(QSystemTrayIcon::ActivationReason)));
     trayIcon->show();
 }
 
+/** Creates a new menubar for macOS */
```

Comment:
same here: ifdef missing, unless this is used on other systems too but up to know we lived without it.

Reply:
i will create a new fresh pr